plugins/vpp/l2plugin/vppcalls/dump_vppcalls.go

```diff
@@ -37,16 +38,22 @@ type BridgeDomainMeta struct {
 }
 
 // DumpBridgeDomains implements bridge domain handler.
-func (handler *BridgeDomainVppHandler) DumpBridgeDomains() (map[uint32]*BridgeDomainDetails, error) {
+func (h *BridgeDomainVppHandler) DumpBridgeDomains() (map[uint32]*BridgeDomainDetails, error) {
 	defer func(t time.Time) {
-		handler.stopwatch.TimeLog(l2ba.BridgeDomainDump{}).LogTimeEntry(time.Since(t))
+		h.stopwatch.TimeLog(l2ba.BridgeDomainDump{}).LogTimeEntry(time.Since(t))
 	}(time.Now())
 
+	// At first prepare bridge domain ARP termination table which needs to be dumped separately
+	bdArpTab, err := h.dumpBridgeDomainMacTable()
+	if err != nil {
+		return nil, fmt.Errorf("failed to dump bridge domains: arp termination table dump returned an error: %v", err)
```

Comment:
The error message seems to be redundant. Shouldn't the part with `failed to dump bridge domains` be coming from the caller? And this would then just be: `arp termination table dump failed: %v"`. Also perhaps this should be created with go-errors package instead?